subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GrettySmokeTest.groovy

```diff
@@ -25,13 +26,13 @@ import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 )
 class GrettySmokeTest extends AbstractPluginValidatingSmokeTest {
 
-    def 'run with jetty'() {
+    def 'run jetty with gretty version'() {
```

Comment:
```suggestion
    def 'run Jetty with Gretty #grettyConfig.version'() {
```

Does this work? If not, you could assign the individual property in the where clause.

Reply:
It does, nice